core/src/main/scala/pink/cozydev/lucille/QueryParser.scala

```diff
@@ -191,17 +191,26 @@ private object Parser {
   }
 
   private val baseRange = (0x20.toChar to 0x10ffff.toChar).toSet
-  private val special = Set('\\', ':', '^', '(', ')', '"', '“', '”', ' ', '*', '?', '~')
+  val luceneSpecial =
+    Set('+', '-', '!', '(', ')', '{', '}', '[', ']', '^', '"', '~', '*', '?', ':', '\\', '/')
+  private val special = Set('“', '”', ' ') ++ luceneSpecial
+  private val escapedTokens = P.char('\\') *> P.charIn(special)
   private val allowed: P[Char] =
     // From cats.parse.strings.Json nonEscaped handling
     P.charIn(baseRange -- special)
   val reserved = Set("OR", "||", "AND", "&&", "NOT", "+", "-", "/")
 
   val queryEnd = (wsp | P.end | P.char(')')).peek
 
-  val term: P[String] = P.not(P.stringIn(reserved)).with1 *> allowed.rep.string
+  // We use repAs[String] to escape the slashes
+  val term: P[String] =
+    P.not(P.stringIn(reserved)).with1 *> allowed.orElse(escapedTokens).repAs[String]
 
-  val phrase: P[String] = (maybeSpace.with1 *> term <* maybeSpace).rep.string.surroundedBy(dquote)
+  private val quotes = Set('"', '“', '”')
+
+  private val phraseTerm = P.charIn(baseRange -- quotes).repAs[String]
```

Comment:
Hmm... because we're not using `escapedTokens` in `phraseTerm` I think we're safe to use `.rep.string` instead of `repAs[String]`

```suggestion
  private val phraseTerm = P.charIn(baseRange -- quotes).rep.string
```

Reply:
Actually this brings to mind that we currently can't form a phrase query with quotes in it.
Which, while weird, is technically something we should allow.

I think the following test should pass (it currently fails)
```scala
test("parse phrase query with quotes") {
  val r = parseQ("\"the cat said \\\"meow\\\" loudly\"")
  assertEquals(r, Right(Phrase("""the cat said "meow" loudly""")))
}
```